portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <h1>My Hidden Talents!</h1>
+    <div class="content">
+      <div id="row">
+        <div class="column">
+          <div id="lavendar-background">
```

Comment:
For this section, one idea is removing the <div id="lavendar-background> element and just keeping the div above. You can add e.g. id="clover-section" to the div above that just has the "column" class. Since it's ok to have an element with a class, id, and even multiple classes.

I suggested clover-section because it keeps the id name more related to the structure of the content, rather than the styling. Then later if we come back and happen to want this section to no longer be lavender, we wouldn't have to change the id name. Notice how currently the id name is pretty much a diguised inline style.

And the same goes with the sections below.

Reply:
Thank you! This actually not only makes the naming more useful, but also makes the display columns more even.